packages/react-components/src/button/src/TextButton.css

```diff
@@ -0,0 +1,111 @@
+:root {
+    --o-ui-button-rounded-padding-2xs: var(--scale-bravo);
+    --o-ui-button-rounded-padding-xs: var(--scale-charlie);
+    --o-ui-button-rounded-padding-sm: var(--scale-delta);
+    --o-ui-button-rounded-padding-md: var(--scale-echo);
+    --o-ui-button-rounded-padding-lg: var(--scale-foxtrot);
+    --o-ui-button-pill-padding-2xs: var(--scale-alpha);
+    --o-ui-button-pill-padding-xs: var(--scale-alpha);
+    --o-ui-button-pill-padding-sm: var(--scale-charlie);
+    --o-ui-button-pill-padding-md: var(--scale-delta);
+    --o-ui-button-pill-padding-lg: var(--scale-echo);
+    --o-ui-button-pill-padding-ratio-factor: 1.5;
```

Comment:
Rename text button var to include `text-button`.

`--o-ui-button-pill-padding-lg` -> `--o-ui-text-button-pill-padding-lg`